src/features/projectsV2/ProjectDetails/components/microComponents/SampleSpecies.tsx

```diff
@@ -0,0 +1,45 @@
+import { SamplePlantLocation } from '../../../../common/types/plantLocation';
+import styles from '../../styles/PlantLocationInfo.module.scss';
+import { useTranslations } from 'next-intl';
+
+interface Props {
+  sampleInterventions: SamplePlantLocation[];
+}
+const SampleSpecies = ({ sampleInterventions }: Props) => {
+  const tProjectDetails = useTranslations('ProjectDetails');
```

Comment:
I feel `SampleTrees` or `SampleInterventions` or `SampleInterventionsList` reflects the component purpose better.

Update class names etc. within the internal structure of this component accordingly.